pkg/deploy/devconfig.go

```diff
@@ -156,6 +156,7 @@ func DevConfig(_env env.Core) (*Config, error) {
 				"InsecureSkipVerifyDBTokenCertificate",
 			},
 			GatewayMDSDConfigVersion:    to.StringPtr("3.3"),
+			GatewayVMSize:               to.StringPtr("Standard_D4s_v3"),
```

Comment:
We have this as a default value in `pkg/deploy/generator/templates_gateway.go` so it is not necessary, but I don't mind being explicit.